web/templates/settings/notification_settings.hbs

```diff
@@ -106,7 +106,11 @@
             </h3>
             {{> settings_save_discard_widget section_name="desktop-message-settings" show_only_indicator=(not for_realm_settings) }}
         </div>
-
+        <div class="tip desktop-setting-notications">
+            {{#tr}}Zulip needs your permission to <z-link>enable desktop notifications.</z-link>
+                {{#*inline "z-link"}}<a class="request-desktop-notifications alert-link" role="button" tabindex=0>{{> @partial-block}}</a>{{/inline}}
```

Comment:
Blue box outline appears on clicking the link, which I think we do not show at other places. So, would be good to check it and remove it here as well if it is not shown in other places.